src/mode.js

```diff
@@ -13,16 +13,25 @@ const chromaticNotes = ['c', 'c#', 'd', 'd#', 'e', 'f', 'f#', 'g', 'g#', 'a', 'a
  * @return {[type]}                        Returns the mode as an array, e.g. ['c3', 'd3', 'e3', 'f3', 'g3', 'a3', 'b3', 'c4']
  */
 const mode = (root, mode, octave, addRootFromNextOctave) => {
-	root = root || 'c';
-	mode = mode || 'ionian';
-	octave = octave ? Number(octave) : 3;
-	addRootFromNextOctave = addRootFromNextOctave === false ? false : true;
+	if (root.includes('|')) {
+		var args = root.split('|');
+		root = args[0];
+		mode = args[1];
+		octave = args[2];
+		addRootFromNextOctave = args[3] == 'false' ? false : true;
```

Comment:
I think we can stick to ===. If you move this to the beginning of the function as such, 
`addRootFromNextOctave = addRootFromNextOctave || true;`, then you can just check if args[3] is 'false' here, coz it will have to be a string if it exists.